documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -11,7 +11,7 @@
 This is the reference documentation of MapStruct, an annotation processor for generating type-safe, performant and dependency-free bean mapping code.
 This guide covers all the functionality provided by MapStruct. In case this guide doesn't answer all your questions just join the MapStruct https://groups.google.com/forum/?fromgroups#!forum/mapstruct-users[Google group] to get help.
 
-You found a typo or other error in this guide? Please let us know by opening an issue in the https://github.com/mapstruct/mapstruct[MapStruct GitHub repostory],
+You found a typo or other error in this guide? Please let us know by opening an issue in the https://github.com/mapstruct/mapstruct[MapStruct GitHub repository],
```

Comment:
Interestingly enough, this typo has persisted since 2015 (or v1.0.0-final)! So many people either didn't care enough to change it or didn't notice it at all since it was such a small error.

Just glad to help out :)